packages/story-editor/src/components/panels/shared/media/backgroundAudioPanelContent.js

```diff
@@ -224,12 +233,37 @@ function BackgroundAudioPanelContent({
         size={BUTTON_SIZES.SMALL}
         variant={BUTTON_VARIANTS.RECTANGLE}
       >
-        {captionText}
+        {__('Upload audio captions', 'web-stories')}
       </UploadButton>
     ),
-    [captionText]
+    []
   );
 
+  const options = [
+    {
```

Comment:
```suggestion
    hasUploadMediaAction && {
```

Don't we need this here?